index.php

```diff
@@ -130,6 +132,7 @@
 
 	// add hypha commands and navigation
 	$_cmds[] = '<a href="index/'.$hyphaLanguage.'">'.__('index').'</a>';
+	if ($O_O->isUser()) $_cmds[] = '<a href="help/index">'.__('help').'</a>';
```

Comment:
Putting the help index in the menu will likely let users expect a well-written help text they can read from top to bottom. Since the current help index is more of a developer tool (especially since it shows the raw identifiers), I would certainly not put it in the menu like this.